aws/table_aws_sagemaker_notebook_instance.go

```diff
@@ -249,7 +249,13 @@ func listAwsSageMakerNotebookInstanceTags(ctx context.Context, d *plugin.QueryDa
 		region = matrixRegion.(string)
 	}
 
-	resourceArn := h.Item.(*sagemaker.DescribeNotebookInstanceOutput).NotebookInstanceArn
+	var resourceArn string
+	switch h.Item.(type) {
+	case *sagemaker.NotebookInstanceSummary:
+		resourceArn = *h.Item.(*sagemaker.NotebookInstanceSummary).NotebookInstanceArn
+	case *sagemaker.DescribeNotebookInstanceOutput:
+		resourceArn = *h.Item.(*sagemaker.DescribeNotebookInstanceOutput).NotebookInstanceArn
+	}
```

Comment:
Can we add a separate function for this?
also, do we need the following check for GET call?
https://github.com/turbot/steampipe-plugin-aws/pull/372/files#diff-2ceeebe3d446a630c976a1a76ab24829e58568b0645e3ae8e4aaa5390a19e234R214-R219